### PR TITLE
Bayo izanami opikuda

### PR DESCRIPTION
**Pull request by iampikuda**
#### What does this PR do?
 - This PR serves as my 4th output for checkpoint 1
 #### Description of Task to be completed?
 - A README.md file and a screencast of the proposed template for my frontend
 #### How should this be manually tested?
 - Viewing the screenshots
 #### Any background context you want to provide?
 - n/a
 #### What are the relevant pivotal tracker stories?
 - n/a
 #### Screencast

[Html Js Css Outputs (template)](https://www.youtube.com/watch?v=-eez_mPzFpI "Damisi's Screencast")

 #### Questions:
- n/a

### CODE REVIEW

File: 06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/inverted.ctr.js [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+
+      $scope.noRepeat = (file) => {
+        if (Object.keys($scope.documentWhole).includes($scope.fileName)) {
+          console.log('it contains');
+        } else {
+          console.log("it's empty");
+        }
+      };
+
+      $scope.validateExt = (file) => {
```
Comment (rhenshaw56) [inline 06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/inverted.ctr.js]: `$scope.validateExt` ? what's Ext

File: 06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/index.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+            this.index[this.textArray[word]].push(parseInt(key));
+          };
+        } else {
+          // console.log("nope");
+          continue;
+        }
+      };
+    };
+    console.log(this.index);
+    console.log(this.textArray);
```
Comment (rhenshaw56) [inline 06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/index.js]: here too.

File: 06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/inverted.ctr.js [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+      $scope.indexedFile = '';
+      $scope.fileRange = [];
+      $scope.wordSet;
+      $scope.text = {};
+      let temp = {};
+      let tempo;
+      $scope.documentRange = {};
+
+
+      const inverted_index = new InvertedIndex();
```
Comment (rhenshaw56) [inline 06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/inverted.ctr.js]: i believe camelCasing is the preferred naming style for us.

File: 06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/inverted.ctr.js [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+
+      $scope.noRepeat = (file) => {
+        if (Object.keys($scope.documentWhole).includes($scope.fileName)) {
+          console.log('it contains');
+        } else {
+          console.log("it's empty");
```
Comment (rhenshaw56) [inline 06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/inverted.ctr.js]: Its still there

File: 06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/inverted.ctr.js [hunk 5]
```diff
@@ -0,0 +1,217 @@
...
+
+      const onFileComplete = (response) => {
+
+        $scope.titleObj = {};
+        $scope.textObj = {};
+        // $scope.$apply();
+        // console.log($scope.textObj);
+
+        let count = 1;
+        console.log('whaaaaaaaaaaaaaaat');
```
Comment (rhenshaw56) [inline 06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/inverted.ctr.js]: `LOOOL` what happened here? talk to me bro.

File: 06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/index.js [hunk 6]
```diff
@@ -0,0 +1,53 @@
+class InvertedIndex {
+
+  constructor() {
+    this.textArray = [];
+    this.index = {};
+  };
+
+  transformToSingles(file) {
+    for (let words in file) {
+      file[words] = file[words].replace(/'\w+\s/g, " ").replace(/[.,/#!+$%^&@*?;:'{}=\-_`~()]/g, '').trim().toLowerCase().split(' ')
```
Comment (rhenshaw56) [inline 06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/index.js]: Is your esliint on ?

File: 06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/index.js [hunk 7]
```diff
@@ -0,0 +1,53 @@
+class InvertedIndex {
+
+  constructor() {
+    this.textArray = [];
+    this.index = {};
+  };
+
+  transformToSingles(file) {
```
Comment (rhenshaw56) [inline 06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/index.js]: you don't have jsDocs ?

Comment (rhenshaw56) [reply 06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/index.js]: I believe this PR is just for submission for your front end files and not your `InvertedIndex` class. If so it looks very good, but you can do away with the console.logs.

File: 06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/inverted.ctr.js (hunk 1 above)
Comment (iampikuda) [reply 06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/inverted.ctr.js]: Ext = extension 
I'll clarify that in final build, thank you.

**Comment by andela-cofor** [comment]
I like the fact that you used a screencast for your output. Looks good to me

File: 06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/index.js (hunk 2 above)
Comment (iampikuda) [reply 06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/index.js]: The ghost code and console.logs? I'll remove all that in final build.

File: 06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/inverted.ctr.js (hunk 3 above)
Comment (iampikuda) [reply 06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/inverted.ctr.js]: I didn't know it was for everything. Thanks again. That's helpful.

File: 06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/inverted.ctr.js (hunk 5 above)
Comment (iampikuda) [reply 06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/inverted.ctr.js]: It's just a wandering console.log. She has served her purpose.

File: 06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/index.js (hunk 6 above)
Comment (iampikuda) [reply 06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/index.js]: No it wasn't. I was having serious issues concentrating when there were red lines everywhere. I'll definitely do better in the next CP.

File: 06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/index.js (hunk 7 above)
Comment (iampikuda) [reply 06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/index.js]: Yes I have it and yes I'll include all that before Monday and hopefully by Tuesday, bossman Neji and I will be done scrutinising my work.

**Comment by iampikuda** [comment]
Thanks Chinedu. I felt screenshots would not do it justice.

File: 06-building-a-program-start-to-finish/checking-repository-health/output/README.md [hunk 8]
```diff
@@ -0,0 +1,15 @@
+## Checking Repository Health
+----
+#### Output
+
+- **Using**: `Code Climate` and `Coveralls`
+- **Deliver**: a screenshot of your code coverage and GPA badges in your repo's README file
+
+#### Link to github repo tree
+- [Inverted Index repo health tree](https://github.com/andela-opikuda/Invertedindex/tree/chore/140010193/set-up-travis-coverall-and-codeclimate)
```
Comment (q-ode) [inline 06-building-a-program-start-to-finish/checking-repository-health/output/README.md:9]: Your badges aren't showing properly. Both of them are unknown

File: 06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/README.md [hunk 9]
```diff
@@ -0,0 +1,12 @@
...
+- **Deliver**: a folder containing the template for the proposed frontend of your project
+- **With**: *independently*
+
+### Link to GitHub Repo
+
+- [andela-opikuda/Invertedindex/](https://github.com/andela-opikuda/Invertedindex/ "Visit Damisi's Project Repo")
+
+### Link to Screencast
+
+- [Html Js Css Outputs (template)](https://www.youtube.com/watch?v=-eez_mPzFpI "Damisi's Screencast")
```
Comment (q-ode) [inline 06-building-a-program-start-to-finish/writing-html-js-css-outputs-(template)/output/README.md:12]: Looks good.

**Comment by iampikuda** [comment]
They are working now. Check this updated branch: https://github.com/andela-opikuda/Invertedindex/tree/switch4
Forgive the branch naming convention mishap.

**Pull request closed without merging**